src/main/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/JsonDefaultValueFormatter.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 HM Revenue & Customs
+ * Copyright 2021 HM Revenue & Customs
```

Comment:
Should we remove this class as suggested in the FIXME?

Reply:
@matthewmascord Have created a Jira ticket to do this separately.